lib/sms.js

```diff
@@ -1,5 +1,44 @@
 const ph = require('./plugin-helper')
 const argv = require('minimist')(process.argv.slice(2))
+const _ = require('lodash/fp')
+
+const customSms = require('./custom-sms')
+
+function getPhoneCodeSms (phone, code) {
+  return customSms.getCustomMessage('sms_code')
+    .then(msg => {
+      if (!_.isNil(msg)) {
+        const messageContent = _.replace('#code', code, msg.message)
+        return {
+          toNumber: phone,
+          body: messageContent
+        }
+      }
+
+      return {
+        toNumber: phone,
+        body: `Your cryptomat code: ${code}`
+      }
+    })
+}
+
+function getCashOutReadySms (phone, timestamp) {
+  return customSms.getCustomMessage('cash_out_dispense_ready')
+    .then(msg => {
+      if (!_.isNil(msg)) {
+        const messageContent = _.replace('#timestamp', timestamp, msg.message)
+        return {
+          toNumber: phone,
+          body: messageContent
+        }
+      }
+
+      return {
+        toNumber: phone,
+        body: `Your cash is waiting! Go to the Cryptomat and press Redeem within 24 hours. [${timestamp}]`
+      }
+    })
+}
```

Comment:
Looks good but these functions seem similar enough so we can define a getCustomSms(phone, event, content) that changes according to the use case (cash out ready or code).